src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+
+.stopwatch {
+  position: relative;
+
+  &__border {
+    width: 80vmin;
+    height: 80vmin;
+    border-radius: 50%;
+    border: 1vmin dotted #000;
+  }
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
```

Comment:
The color value `#0700ff` should be written in lowercase as per the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).

Reply:
??